modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+On the **Target details** page, you can view detailed information about a metric target.
```

Comment:
In the preview link, I read the intro and that tells me why I care about Metric Targets: I can view, search, and filter endpoints for scraping. 

For this section, it might help to remind users why they care about getting more information about a target. Something simple, like:

`On the **Target details** page, you can view detailed information about a metric target, helping you identify the endpoints for scraping.` Or maybe something a bit more focused on the advantage for development. 

Note, this is simple a consideration. Your intro here is succinct and very good as is!

Reply:
Those are good points. But I hope that the opening section of content in the assembly file `managing-metrics-targets.adoc` covers the main reason(s) why this feature is useful. This section is pretty close to the beginning, so I feel that adding the same info here would be redundant.